pkg/kubelet/cm/container_manager.go

```diff
@@ -41,13 +41,14 @@ import (
 )
 
 type ActivePodsFunc func() []*v1.Pod
+type TerminatingPodsFunc func() []*v1.Pod
 
 // Manages the containers running on a machine.
 type ContainerManager interface {
 	// Runs the container manager's housekeeping.
 	// - Ensures that the Docker daemon is in a container.
 	// - Creates the system container where all non-containerized processes run.
-	Start(*v1.Node, ActivePodsFunc, config.SourcesReady, status.PodStatusProvider, internalapi.RuntimeService) error
+	Start(*v1.Node, ActivePodsFunc, TerminatingPodsFunc, config.SourcesReady, status.PodStatusProvider, internalapi.RuntimeService) error
```

Comment:
uhm, do we still need ActivePodsFunc? (here and in all the other managers)

Reply:
Yes, for the CPU manager reconcile loop. I did not want to change the reconcile loop to use terminating pods, we definitely can re-visit it lately and improve it.

Reply:
keeping ActivePodsFunc here for this reason seems actually fair to me even in the long run